pkg/resource/aws/aws_dynamodb_table_ext.go

```diff
@@ -0,0 +1,19 @@
+package aws
+
+import "github.com/cloudskiff/driftctl/pkg/resource"
+
+func (r *AwsDynamodbTable) NormalizeForState() (resource.Resource, error) {
+	// set default to empty struct for timeouts
+	if r.Timeouts == nil {
```

Comment:
It's a Terraform only related field, for consistency purpose it's more straightforward to handle it like in other resource by adding a `diff:"-"` annotation.

Reply:
ho you're right I thought these were real timeout not only used by tf...